app/javascript/mastodon/features/follow_requests/index.js

```diff
@@ -59,9 +72,9 @@ export default class FollowRequests extends ImmutablePureComponent {
 
         <ScrollContainer scrollKey='follow_requests'>
           <div className='scrollable' onScroll={this.handleScroll}>
-            {accountIds.map(id =>
+            {this.state.accountIds ? this.state.accountIds.map(id =>
```

Comment:
`{this.state.accountIds ? this.state.accountIds.map(id => {}) : null` -> `{this.state.accountIds && this.state.accountIds.map(id => {})`

Reply:
okay, thanks.